webonary-cloud-api/lambda/searchEntries.ts

```diff
@@ -165,7 +165,7 @@ export async function searchEntries(args: SearchEntriesArguments): Promise<Respo
         .collection(DB_COLLECTION_DICTIONARY_ENTRIES)
         .find(dictionaryPartialSearch)
         .collation({ locale, strength })
-        .sort({ 'mainHeadWord.value': 1, _id: 1 })
+        .sort({ displayText: 1, _id: 1 })
```

Comment:
Does this actually accomplish what was requested?

Reply:
Good question. Clarifying the requirements on https://github.com/sillsdev/webonary/issues/397#issuecomment-1230731343